### DISCUSSION TITLE
External keyboard mimicking laptop keyboard after 0.161.0

### DISCUSSION THREAD

**Discussion by deztech93** [seerge/g-helper#2299] [General]
Is anyone else experiencing an issue like this? my nuphy keyboard function key shortcuts are mimicking the placement of the function key shortcuts on my laptop keyboard despite changing them in nuphy console. Can't figure out solution.

**Comment by seerge** [comment]
@deztech93 you just need to turn ``FN-Lock`` off :) There is nothing changed in this area specifically in 0.161, it has always been like that

**Reply by deztech93** [reply]
If I turn off FN-lock, neither my external keyboard or laptop keys have the function key shortcuts enabled. The issue is that my external keyboard is matching the function key shortcuts of my laptop. For example, the F1 on my 2022 G14 is set to mute and my external keyboard is set to decrease screen brightness but mute is activated when I click on my external keyboard the shortcut that should be decrease brightness. This issue started as soon as the G-Helper update came in so I'm not sure on what else would be causing this odd issue. I've spent a while trying to search for a fix

**Reply by seerge** [reply]
@deztech93 if you turn FN-lock off - it is obviously turned OFF, and you need to press FN + F1 in order to get desired function.

When it's on app just registers F1-F11 as own hotkeys in windows, and runs corresponding actions when they are pressed. This is software FN-lock, as Asus devices don't have hardware FN-lock obviously. 

And it is like that since the very beginning. You are free to download any older version to verify :)
